Dockerfile.pr

```diff
@@ -0,0 +1,69 @@
+# Form viewier that runs as a Lambda function.  It is used to create 
+# Lambda function review environments for each PR.
+FROM node:16@sha256:2ed6bba040f90005db9785927689c0d9a9442ca2cf9a59dc52297d684285c094 as build
+
+ENV NODE_ENV=production
+ENV NEXT_OUTPUT_STANDALONE=true
+
+COPY . /src
+WORKDIR /src
+
+ARG GOOGLE_CLIENT_ID
```

Comment:
Just a heads up that the GOOGLE_CLIENT_ID and SECRET are no longer used in the app.

Reply:
ah good stuff - I've removed them both